### DISCUSSION TITLE
How can I open the URL via lighthouse in the existing chrome incognito window

### DISCUSSION THREAD

**Discussion by ankur-lt** [GoogleChrome/lighthouse#15142] [General]
I am opening chrome using the folloiwng command:
```sh
/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --remote-debugging-port=9223
```
I opened the browser and opened its incognito window, closed the normal window.

Now I run the following command to get the lighthouse report:
```
lighthouse https://www.google.com --port=9223 --chrome-flags="--incognito"
```
**Result:** This command is opening a normal chrome window and opening the URL in its tab.
**Expectation:** The command should open the URL in the existing chrome incognito window.

Am I missing something in the CLI command?

**Comment by connorjclark** [comment]
For what purpose are you wanting to open an incognito Chrome? AFAIK, all incognito does is open Chrome with a fresh profile, which is what the lighthouse CLI does by default when tasked with opening Chrome (when you provide a debug port, you are tasked with providing Chrome in whatever state you want it to be in).

**Reply by ankur-lt** [reply]
I am using Playwright library to run my UI tests. It opens the browser by default in incognito mode. I want to run the above given lighthouse command with the same incognito browser context while the test is in progress in an another tab, so that my website's auth is maintained in the browser context.
